tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     transaction
     zc.lockfile
     zdaemon
+    zodbpickle
```

Comment:
I believe I was the one who added that comment originally.  ;-)

Not necessarily in this particular tox.ini; things like that tend to be cargo-culted around.